docs/plan/dashboard-golem-redesign.md

```
@@ -0,0 +1,428 @@
+
+---
+
+## Branch
+
+Create branch `feature/dashboard-golem-redesign` from master.
+
+---
+
+## File Structure After Changes
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)